Kajabi/sage-lib
# [DSS-420] Correct icons in Alert React component

## Description
DSS-420

Icons displayed for the Alert warning/approaching and danger/exceeded React component are reversed from the designs in [Figma](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/%F0%9F%A7%A9-Sage-components?type=design&node-id=21491-34243&t=c3qONTcX0uh95PtB-4).


## Screenshots

|  | Before  |  After  |
|-----|---|--------|
| Warning |![warning-before](https://github.com/Kajabi/sage-lib/assets/816579/657a3e96-444f-4ba9-abf4-19ad75edc01d)|![warning-after](https://github.com/Kajabi/sage-lib/assets/816579/d2ba2f48-7492-4383-93c1-b0b27e613161)|
| Danger |![danger-before](https://github.com/Kajabi/sage-lib/assets/816579/5abc5ce0-447e-449d-b781-ee1de270ce2d)|![danger-after](https://github.com/Kajabi/sage-lib/assets/816579/94e32050-7fe5-49f4-8a40-eb4dc25329b6)|


## Testing in `sage-lib`
1. Navigate to the Alert component in Storybook
2. Select the "Warning" and "Danger" variants from the `color` dropdown
3. Confirm that the icons match the Figma spec and Rails component


## Testing in `kajabi-products`
1. (**HIGH**) Alert - icons updated for "Warning" and "Danger" React component. Style-only; no anticipated change in functionality
   - [ ] Payments Integrations -> Kajabi Payments 2FA alert
   - [ ] Custom email domain (CEDS) upgrade alert
